packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/nlb/network-listener.ts

```diff
@@ -222,6 +229,20 @@ export class NetworkListener extends BaseListener implements INetworkListener {
     if (props.defaultTargetGroups) {
       this.setDefaultAction(NetworkListenerAction.forward(props.defaultTargetGroups));
     }
+
+    if (props.tcpIdleTimeout !== undefined && !Token.isUnresolved(props.tcpIdleTimeout)) {
```

Comment:
I think it is better to handle the case where `tcpIdleTimeout` is smaller than 1 sec.

```suggestion
    if (props.tcpIdleTimeout !== undefined && !Token.isUnresolved(props.tcpIdleTimeout)) { 
     if (props.tcpIdleTimeout.toMilliseconds() < Duration.seconds(1).toMilliseconds()) {
       throw new Error(`\`tcpIdleTimeout\`` must be between 60 and 6000 seconds, got ${props.tcpIdleTimeout.toMilliseconds()} milliseconds.`)
     }
```

And could you please add unit test about this?